Objects/Objects/BuiltElements/Revit/FamilyInstance.cs

```diff
@@ -1,60 +1,67 @@
-﻿using System;
-using Objects.Geometry;
+﻿using Objects.Geometry;
 using Objects.Utils;
 using Speckle.Core.Kits;
 using Speckle.Core.Models;
+using Speckle.Newtonsoft.Json;
+using System;
 using System.Collections.Generic;
 using System.Linq;
-using Speckle.Newtonsoft.Json;
 
 namespace Objects.BuiltElements.Revit
 {
   public class FamilyInstance : Base, IDisplayMesh, IDisplayValue<List<Mesh>>
   {
-    public Point basePoint { get; set; }
     public string family { get; set; }
     public string type { get; set; }
     public string category { get; set; }
+    public int categoryId { get; set; }
+    public string elementId { get; set; }
+    public Point basePoint { get; set; }
     public Level level { get; set; }
+    public Orientation orientation { get; set; }
     public double rotation { get; set; }
     public bool facingFlipped { get; set; }
     public bool handFlipped { get; set; }
     public bool mirrored { get; set; }
     public Base parameters { get; set; }
-    public string elementId { get; set; }
 
     [DetachProperty]
     public List<Base> elements { get; set; }
-    
+
     [DetachProperty]
     public List<Mesh> displayValue { get; set; }
 
     public string units { get; set; }
-
     public FamilyInstance() { }
 
     [SchemaInfo("FamilyInstance", "Creates a Revit family instance", "Revit", "Families")]
-    public FamilyInstance(Point basePoint, string family, string type, Level level,
```

Comment:
A new constructor needs to be added with the new arguments to not break backwards compatibility with existing GH components